### PR TITLE
Add demo for max downscaling image

### PR DESCRIPTION
**Pull request by loonybear**

### CODE REVIEW

File: public/demos/max-downscaling-image.html [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+<details class="details">
+  <!-- filled dynamically -->
+</details>
+
+<h2 id="feature-allowed-banner">
+  <span id="allowfeature">...</span>
+</h2>
+
+<p>Click the disallow/allow buttons above to see how the page loads with the policy on and off.
+  When the policy is on, you will see images whose intrinsic dimentions that are much larger
```
Comment (ebidel) [inline public/demos/max-downscaling-image.html]: When the policy is on, you will see images that have intrinsic dimensions which are much larger than their container size.

File: public/demos/max-downscaling-image.html [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+  than the container size.</p>
+
+<!--
+For optimized images:
+1. max-downscaling-image on: image will be rendered accurately.
+2. max-downscaling-image off: image will be rendered accurately.
```
Comment (ebidel) [inline public/demos/max-downscaling-image.html]: rendered accurately -> rendered normally

File: public/demos/max-downscaling-image.html [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+</div>
+
+<script type="module">
+import * as shared from '/js/shared.js';
+
+shared.initPage();
+
+// https://chromium.googlesource.com/chromium/src.git/+/832440db5c586d1bdeedfd01489e43d09de36aa9
+// console.log('Allowed features:', document.policy.allowedFeatures());
+//document.policy.allowsFeature('unsized-images')
```
Comment (ebidel) [inline public/demos/max-downscaling-image.html]: can remove these comments

File: public/js/policies.json [hunk 4]
```diff
@@ -102,4 +102,22 @@
...
+  "url": "/demos/image-downscaling.html",
+  "what": "Restricts images whose intrinsic dimensions that are much larger than the container size.",
+  "why": "Ensures optimized image performance by avoiding unnecessarily large images.",
+  "usage": {
+    "off": "max-downscaling-image 'self'",
+    "on": "max-downscaling-image 'none'"
+  },
+  "examples": {
+    "header": [
+      "FeaturePolicy: max-downscaling-image 'none'"
```
Comment (ebidel) [inline public/js/policies.json]: Feature-Policy

File: public/js/policies.json [hunk 5]
```diff
@@ -102,4 +102,22 @@
       "&lt;iframe allow=\"animations https:\/\/anim8.cdn.com\" src=\"...\"><\/iframe>"
     ]
   }
+}, {
+  "id": "max-downscaling-image",
+  "name": "Maximum Downscaling Image",
+  "url": "/demos/image-downscaling.html",
+  "what": "Restricts images whose intrinsic dimensions that are much larger than the container size.",
```
Comment (ebidel) [inline public/js/policies.json]: Warns when images with intrinsic dimensions are much larger than their container size.

File: public/js/policies.json [hunk 6]
```diff
@@ -102,4 +102,22 @@
...
+  "why": "Ensures optimized image performance by avoiding unnecessarily large images.",
```
Comment (ebidel) [inline public/js/policies.json]: Image bloat is a large problem on the web. Sending unnecessarily large images is bad for performance, UX, and wastes bandwidth.

**Comment by loonybear** [comment]
Done. Please take another look. Thanks

File: public/demos/max-downscaling-image.html [hunk 7]
```diff
@@ -20,17 +20,17 @@ <h2 id="feature-allowed-banner">
 </h2>
 
 <p>Click the disallow/allow buttons above to see how the page loads with the policy on and off.
-  When the policy is on, you will see images whose intrinsic dimentions that are much larger
-  than the container size.</p>
+  When the policy is on, you will see images that have intrinsic dimentions which are much larger
+  than their container size render with inverted colors..</p>
```
Comment (ebidel) [inline public/demos/max-downscaling-image.html]: Double ..

File: public/js/policies.json (hunk 5 above)
Comment (ebidel) [reply public/js/policies.json]: Can you update the language to ^?

**Comment by ebidel** [comment]
This will need a rebase.

**Comment by loonybear** [comment]
Done. Thanks.

**Pull request merged**